README.md

```diff
@@ -355,12 +355,12 @@ The `Watcher` interface defined above gives the basis for implementing typical J
 // NOTE: This scheduling logic is too basic to be useful. Do not copy/paste.
 let pending = false;
 
-let w = new Signal.subtle.Watcher(() => {
+const w = new Signal.subtle.Watcher(() => {
     if (!pending) {
         pending = true;
         queueMicrotask(() => {
             pending = false;
-            for (let s of w.getPending()) s.get();
+            for (let s of w.getPending()) { s.get(); }
```

Comment:
Maybe we should make it a multi-line `for` then, wdyt ?

Reply:
I absolutely agree, if it's possible. I didn't know what the goals were, so in case one of them was "keeping line count as low as possible", I didn't want to interfere. In my opinion this is longer but leaves no room for questions:
```suggestion
            for (let s of w.getPending()) {
                        s.get();
            }
```
Should I update PR?